addons/udes_stock/data/picking_types.xml

```diff
@@ -76,8 +76,8 @@
       <field name="code">internal</field>
       <field name="sequence_id" ref="sequence_internal"/>
       <field name="use_existing_lots" eval="1"/>
-      <field name="default_location_src_id" ref="stock.stock_location_stock"/>
-      <field name="default_location_dest_id" ref="stock.stock_location_stock"/>
+      <field name="default_location_src_id" ref="stock.warehouse0_view_location"/>
```

Comment:
Question: Story says about rockhopper, do we want by default in every possible package or future customer, or the change should be done in udes_rockhopper module.

The change looks good

Reply:
I think this is fine since we want both adhoc and planned to behave the same so the answer is to have both act the same by default.